src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,34 @@
 package core.basesyntax;
 
 public class MainApp {
+    public MainApp(String[] machine) {
```

Comment:
The constructor for `MainApp` should not have parameters since it's not being used. Consider removing the parameter `String[] machine` if it's not necessary for your application logic.